tests/bluetooth/bsim_bt/bsim_test_mesh/prj.conf

```diff
@@ -44,3 +44,9 @@ CONFIG_BT_MESH_PROV_OOB_PUBLIC_KEY=y
 CONFIG_BT_MESH_MODEL_EXTENSIONS=y
 
 CONFIG_BT_MESH_DEBUG=y
+
+# Replace the legacy advertiser with the extended one due to a bug in simulation
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=1
+CONFIG_BT_MESH_ADV_EXT=y
```

Comment:
> When using the legacy advertiser, I see more messages in the scan
callback on a receiver device than were supposed to be sent out from a
sender device even if the network transmit count state is zero.

This is a side effect of the inaccuracy of the legacy advertiser mechanism, and can't be fixed. Could be fixed in the test by ignoring duplicate advertisements, I think

Reply:
They can't be ignored at the top and affects on test's measurements. I will have to increase jitter margin on a receiver.

Reply:
Wouldn't it be possible to ignore raw byte-for-byte duplicates? Both access and transport repeats will have different seqnums